.github/workflows/codeql.yaml

```diff
@@ -1,16 +1,28 @@
 name: "CodeQL"
+
 on:
-  workflow_dispatch: {}
+  push:
+    branches: ["master", "release-*", "!*-merge-master"]
```

Comment:
is it needed? `"!*-merge-master"`, also should we check this only on master?

Reply:
Can we just change the job to create a bracnh `release-*/merge-master` or `merge-release-*-to-master`. The bad naming of the branch is the problem here

Reply:
Removed this from the current workflow and opened https://github.com/kumahq/kuma/issues/12507 to follow up with the mentioned change